SmartDeviceLink/SDLResponseDispatcher.m

```diff
@@ -112,6 +112,7 @@ - (void)storeRequest:(SDLRPCRequest *)request handler:(nullable SDLResponseHandl
         SDLScrollableMessage *scrollableMessage = (SDLScrollableMessage *)request;
         [self sdl_addToCustomButtonHandlerMap:scrollableMessage.softButtons];
     } else if ([request isKindOfClass:[SDLShow class]]) {
+        [self sdl_removeOldButtonHandlers];
```

Comment:
Remember that `Show`s are not the only place with soft buttons, Alerts and ScrollableMessages do as well.